src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +55,34 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService =
+                (MovieSessionService) injector.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        AuthenticationService authenticationService =
+                (AuthenticationService) injector.getInstance(AuthenticationService.class);
+        User user = new User();
+        user.setEmail("12345");
+        user.setPassword("12345_password");
+        User registered;
+        try {
+            registered = authenticationService.register(user.getEmail(), user.getPassword());
+        } catch (RegistrationException e) {
+            throw new RuntimeException(e);
+        }
+
+        ShoppingCartService shoppingCartService =
+                (ShoppingCartService) injector.getInstance(ShoppingCartService.class);
+        ShoppingCart byUser = shoppingCartService.getByUser(registered);
+
+        OrderService orderService =
+                (OrderService) injector.getInstance(OrderService.class);
+        orderService.completeOrder(byUser);
```

Comment:
You are only calling methods from OrderService and not using a result. How about adding some methods that would check if the called methods behave correctly? For example assign orderService.completeOrder(byUser) to a variable and then do even a simple check if it's not null or print it. The same for method getOrdersHistory() - check if list is not null or print it.